package.json

```diff
@@ -172,6 +172,7 @@
 		"fixtures:server-registered": "wp-scripts env docker-run php ./bin/get-server-blocks.php > test/integration/full-content/server-registered.json",
 		"fixtures:generate": "npm run fixtures:server-registered && cross-env GENERATE_MISSING_FIXTURES=y npm run test-unit",
 		"fixtures:regenerate": "npm run fixtures:clean && npm run fixtures:generate",
+		"format-js": "wp-scripts format-js",
```

Comment:
Yes, it would be nice to add a formatting pre-commit step with `lint-staged`. I didn't add that to the original PR, but can add it now if we agree about the desired workflow.